server/app/views/v1/domain_authorizations/_domain_authorization.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+json.domain authorization.domain
+json.challenge authorization.challenge
+json.challenge_opts authorization.challenge_opts
+json.authorization_type authorization.authorization_type
+json.linked_service authorization.grid_service.to_path if authorization.grid_service
+json.service_deploy_id authorization.service_deploy_id if authorization.service_deploy_id
```

Comment:
Consider replacing this with some status/error computed from the `linked_service_deploy`, hiding the service deploy from the API?

Say we find that redeploying the LB is too disruptive, and we implement some fancy etcd-backed challenge store handled by the new LB manager.